spec/serializers/rest/notification_group_serializer_spec.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe REST::NotificationGroupSerializer do
+  subject do
+    serialized_record_json(
+      notification_group,
+      described_class
+    )
+  end
+
+  let(:notification_group) { Fabricate(:notification_group) }
+
+  context 'when latest_page_notification_at is populated', pending: 'no notification_group fabricator' do
```

Comment:
For this one - there is a `NotificationGroup` class, but its not a db-backed AR model ... it is also a serializer. Even more nuanced, it relies on a `pagination_data` object to be present to even return the relevant datetime.

Bare minimum might be something like...

```
mastodon(dev)> ng = NotificationGroup.new pagination_data: { latest_notification_at: 3.days.ago }, notification: Fabricate(:notification), sample_accounts: []
mastodon(dev)> REST::NotificationGroupSerializer.new(ng).to_json
```

Reply:
That works. Thanks.